MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -424,6 +424,11 @@ public string GetAudioEncoder(EncodingJobInfo state)
 
             if (string.Equals(codec, "aac", StringComparison.OrdinalIgnoreCase))
             {
+                // Prefer to use libfdk_aac for better audio quality while using the custom build FFmpeg
+                if (_mediaEncoder.SupportsEncoder("libfdk_aac"))
+                {
+                    return "libfdk_aac";
+                }
                 return "aac -strict experimental";
```

Comment:
I wonder if we should drop `-strict experimental`, I think it's from pre-3.0 era or something...

Reply:
`-strict experimental` or `-strict -2` was previously required for the `aac` encoder, but it is no longer experimental and these options are unnecessary since 5 December 2015.

http://git.videolan.org/?p=ffmpeg.git;a=commit;h=d9791a8656b5580756d5b7ecc315057e8cd4255e

If we don't consider backward compatibility with ffmpeg, we can remove it.

Reply:
I think we're now requiring at least ffmpeg 4.0, which was released in 2018. Seems safe to drop that visual clutter :)

Reply:
```suggestion
                return "aac";
```